tests/base/src/test/java/org/wildfly/security/sasl/otp/OTPTest.java

```diff
@@ -1082,6 +994,40 @@ private CallbackHandler createClientCallbackHandler(String username, String pass
         return ClientUtils.getCallbackHandler(new URI("remote://localhost"), context);
     }
 
+    private void testAuthentication(String seed, int sequenceNumber) throws Exception {
```

Comment:
Would be good to rename this to explain what it does, e.g., checkPasswordRemainsUnchanged or testPasswordRemainsUnchanged, etc.